com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/SerializedHDRenderPipelineGlobalSettings.cs

```diff
@@ -92,14 +92,14 @@ public SerializedHDRenderPipelineGlobalSettings(SerializedObject serializedObjec
 
             renderPipelineResources = serializedObject.FindProperty("m_RenderPipelineResources");
             renderPipelineRayTracingResources = serializedObject.FindProperty("m_RenderPipelineRayTracingResources");
-            defaultFrameSettings = new SerializedFrameSettings(serializedObject.FindProperty("m_RenderingPathDefaultCameraFrameSettings"), null); //no overrides in HDRPAsset
+            defaultCameraFrameSettings = new SerializedFrameSettings(serializedObject.FindProperty("m_RenderingPathDefaultCameraFrameSettings"), null); //no overrides in HDRPAsset
             defaultBakedOrCustomReflectionFrameSettings = new SerializedFrameSettings(serializedObject.FindProperty("m_RenderingPathDefaultBakedOrCustomReflectionFrameSettings"), null); //no overrides in HDRPAsset
             defaultRealtimeReflectionFrameSettings = new SerializedFrameSettings(serializedObject.FindProperty("m_RenderingPathDefaultRealtimeReflectionFrameSettings"), null); //no overrides in HDRPAsset
 
             InitializeCustomPostProcessesLists();
 
             volumeProfileDefault  = serializedObject.FindProperty("m_VolumeProfileDefault");
-            volumeProfileLookDev  = serializedObject.FindProperty("m_VolumeProfileLookDev");
+            lookDevVolumeProfile  = serializedObject.FindProperty("m_LookDevVolumeProfile");
```

Comment:
ha sorry but now we have another name inconsistency here:
**LookDev**VolumeProfile
VolumeProfile**Default**

Should this be defaultVolumeProfile?

Reply:
Is there any release with this version landed?
yes -> yes you need to
no -> just change it quickly before any release land

Reply:
those changes landed in srp2core so i did a migration pass in updated file